### DISCUSSION TITLE
[Feature Request] Add Date and/or Time Picker and ColorPicker components.

### DISCUSSION THREAD

**Discussion by MiyamuraMiyako** [heroui-inc/heroui#1527] [Feature requests]
### Is your feature request related to a problem? Please describe.

The date and time picker makes it easy to set the date and time, and the color picker makes it easy to quickly select colors.

### Describe the solution you'd like

None

### Describe alternatives you've considered

no good alternative.

### Screenshots or Videos

![图片](https://github.com/nextui-org/nextui/assets/10345361/b1f12965-22c4-46ef-b081-0244591003e6)
![图片](https://github.com/nextui-org/nextui/assets/10345361/7568dd16-879b-440c-9faf-2408a3e0004f)

![图片](https://github.com/nextui-org/nextui/assets/10345361/75dac4a9-aa63-4f91-ba64-60b8c14e3768)

**Comment by karkkijokeri** [comment]
I think this would be very useful

**Reply by ted-marozzi** [reply]
I would really like a colour picker
